### ISSUE TITLE
failed to find Changes in the distribution

### ISSUE THREAD

**Issue by eserte** [HayoBaan/Dist-Zilla-PluginBundle-Author-HAYOBAAN#2]
Tests currently fail, maybe only with newer Dist::Zilla versions:

```
PERL_DL_NONLAZY=1 "/usr/perl5.24.0p/bin/perl5.24.0" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/00-compile.t .. ok
[@Author::HAYOBAAN/NextRelease] failed to find Changes in the distribution at /usr/perl5.24.0p/lib/site_perl/5.24.0/amd64-freebsd/Moose/Meta/Method/Delegation.pm line 110.
t/hayobaan.t .... 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
```

**Comment by HayoBaan** [comment]
Hi Slaven,

Thanks for the heads up, the problem was indeed related to change in the 6.005 and later releases of Dist::Zilla. I have fixed it and uploaded the updated version to CPAN just now.

**Comment by bluetyson** [comment]
I just got this error...Dist::Zilla is 6.007 downloaded a few hours ago.

**Comment by HayoBaan** [comment]
Have you updated Dist::Zilla::PluginBundle::Author::HAYOBAAN to v0.011 already? The new version should have fixed the error with Dist::Zilla 6.005 and later.
